web/src/app/pages/create-survey/create-survey.component.html

```diff
@@ -13,23 +13,22 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<div>
-  <h1>Start building your survey</h1>
-  <form [formGroup]="formGroup">
-    <mat-form-field class="input" appearance="fill">
-      <mat-label>
-        Title
-      </mat-label>
-      <input matInput [formControlName]="titleControlKey" placeholder="Survey title" />
-    </mat-form-field>
 
-    <mat-form-field class="input" appearance="fill">
-      <mat-label>
-        Description
-      </mat-label>
-      <input matInput [formControlName]="descriptionControlKey" placeholder="Survey description" />
-    </mat-form-field>
-  </form>
-  <button id="back-button" mat-stroked-button (click)="back()">Back</button>
-  <button id="continue-button" mat-flat-button color="primary" (click)="continue()">Continue</button>
-</div>
\ No newline at end of file
+<div class="container">
+  <div class="card">
+    <name-survey
+      *ngIf="isCreateSurvey()"
```

Comment:
I'd strongly prefer a separate top-level component for create and edit. The layout, primarily the page chrome, will be different. We can avoid a lot of conditional branching by having separate top-level pages.

Reply:
Yes, this component is for create page only, there will be another one for edit, and when adding that page I'll redirect this page to edit page if survey is in `DONE` state.